src/modules/units/components/UnitManagementTable.tsx

```diff
@@ -110,51 +58,170 @@ const UnitManagementTable = ({
         key: 'clients',
         render: (unit) => <UnitOccupants unit={unit} />,
       },
-      ...(allowDeleteUnits
+      ...(canViewCoordinatedEntry
         ? [
             {
-              key: 'delete',
-              width: '1%',
-              dontLink: true,
-              render: (unit: UnitFieldsFragment) =>
-                renderDeleteButton([unit.id], unit.occupants.length > 0),
+              header: 'Accepting Referrals?',
+              key: 'available',
+              render: (unit: UnitFieldsFragment) => {
+                if (
+                  unit.latestOpportunity &&
+                  !unit.latestOpportunity.referral &&
+                  unit.occupants.length === 0
+                ) {
```

Comment:
This seems like logic that could be moved to the backend. (eg `accepting_referrals` on the unit). That might make things easier for testing and to have this "status" displayed consistently throughout the application.